src/firefly/java/edu/caltech/ipac/firefly/server/visualize/WebPlotReader.java

```diff
@@ -401,5 +406,48 @@ private static double getImageSize(WebPlotRequest r) {
         }
     }
 
+    private static List<List<RelatedData>> investigateRelations(File f, FitsRead frAry[]) {
+        if (frAry.length>1) {
+            int imageIdx= -1;
+            List<List<RelatedData>> retList= new ArrayList<>(frAry.length);
+            for(int i=0; (i<frAry.length); i++) {
+                retList.add(null);
+                String extType= frAry[i].getExtType();
+                if (extType!=null && extType.equalsIgnoreCase("IMAGE")) {
+                    imageIdx = i;
+                }
+            }
```

Comment:
I will add some. This function could be there start of something far more advanced.  Right now the way I have it is that the last image with the header 'EXTTYPE' == 'IMAGE' is the based.  I reality that works out to the the first extension.